charts/incubator/amd-automatic-music-downloader/values.yaml

```diff
@@ -0,0 +1,37 @@
+image:
+  repository: tccr.io/truecharts/automatic-music-downloader
+  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
we never tag latest, always use digest-pinned fixed tags when possible, but at least always use digest-pinning